library/common/types/utility.h

```diff
@@ -0,0 +1,13 @@
+#pragma once
+
+#include <string>
+
+#include "library/common/types/c_types.h"
+
+namespace Envoy {
+namespace Types {
+
+envoy_map makeEnvoyMap(std::vector<std::pair<std::string, std::string>> pairs);
```

Comment:
this method probably belong here: https://github.com/envoyproxy/envoy-mobile/blob/main/library/common/data/utility.h

Reply:
mhm, this sounds like a good idea - @goaway mind chiming in since I talked with you about the current location of this file (which I know is not perfect)? I think that `common/data/utility` may be a good place for my utility function too - do you have any preference here?

Reply:
I actually think the Data namespace might be overspecified. I agree there's an opportunity for cleanup, but I'd suggest limiting the scope of this PR. (I think a Bridge namespace alongside these type definitions might make more sense moving forward, and have been thinking a little bit about how to restructure this layer in general.)

Reply:
Moved to Bridge named  in https://github.com/envoyproxy/envoy-mobile/pull/1602/commits/2f00eae81101ae1b94c16e5f15ee84d78bfed3a5